### ISSUE TITLE
uncaught exception

### ISSUE THREAD

**Issue by r569594043** [stnvh/gulp-tinypng-compress#13]
```
[15:59:03] Using gulpfile E:\Git\template-generator\gulpfile.js
[15:59:04] Starting 'tinypng'...
[15:59:07] gulp-tinypng-compress [compressing] ✔ desc-2.png (done)
[15:59:08] gulp-tinypng-compress [compressing] ✔ desc-1.png (done)
[15:59:09] gulp-tinypng-compress [compressing] ✔ desc-3.png (done)
[15:59:16] gulp-tinypng-compress [compressing] ✔ pic-1.jpg (done)
E:\Git\template-generator\node_modules\gulp-tinypng-compress\index.js:176
                            count: res.headers['compression-count'] || 0
                                      ^

TypeError: Cannot read property 'headers' of undefined
    at Object.<anonymous> (E:\Git\template-generator\node_modules\gulp-tinypng-compress\index.js:176:39)
    at self.callback (E:\Git\template-generator\node_modules\gulp-tinypng-compress\node_modules\request\request.js:368:22)
    at emitOne (events.js:77:13)
    at Request.emit (events.js:169:7)
    at Request.onRequestError (E:\Git\template-generator\node_modules\gulp-tinypng-compress\node_modules\request\request.js:1025:8)
    at emitOne (events.js:77:13)
    at ClientRequest.emit (events.js:169:7)
    at TLSSocket.socketErrorListener (_http_client.js:259:9)
    at emitOne (events.js:77:13)
    at TLSSocket.emit (events.js:169:7)
```

**Comment by stnvh** [comment]
I'll add better error handling, thanks for pointing this out.

**Comment by daiqing2009** [comment]
This problem is very tedious, please do fix it

**Comment by stnvh** [comment]
This should be fixed as of 1.2.1
